Scripts/prepare_release.py

```diff
@@ -290,10 +290,16 @@ def _head_up_to_date(cls) -> bool:
     def _no_release_blocking_issues() -> bool:
         progress("Checking... ", end="")
         HEADERS = {"Accept": "application/vnd.github.v3+json"}
-        ENDPOINT = 'https://api.github.com/repos/dafny-lang/dafny/issues?labels=severity%3A+release-blocker'
-        with urlopen(Request(ENDPOINT, data=None, headers=HEADERS)) as fs:
-            response = fs.read().decode("utf-8")
-        return json.loads(response) == []
+        ENDPOINTS = (
+            'https://api.github.com/repos/dafny-lang/dafny/issues?labels=severity%3A+release-blocker',
+            "https://api.github.com/repos/dafny-lang/ide-vscode/issues?labels=severity%3A+release-blocker",
```

Comment:
```suggestion
            'https://api.github.com/repos/dafny-lang/ide-vscode/issues?labels=severity%3A+release-blocker',
```

Reply:
I actually made the opposite change (single quote to double quote, on the other endpoint) for consistency with the rest of the file.